test/e2e/specs/fakerp/admin.go

```diff
@@ -1,23 +1,33 @@
-//+build e2e
-
-package e2e
+package fakerp
 
 import (
 	"strings"
 	"time"
 
-	"github.com/ghodss/yaml"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/ghodss/yaml"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/openshift/openshift-azure/pkg/jsonpath"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
 )
 
 var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader]", func() {
```

Comment:
Why is this spec part of the fakerp directory?

Reply:
It is reliant on a client which cannot be got when running with the production RP.  Therefore it can only be run against the fake RP.  As discussed, hopefully this will improve in the future.